decidim/decidim
# Fix disabling access to space private users when the space is public

#### :tophat: What? Why?
This PR fixes the private user menu entry issue described in #9854

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #9854

#### Testing
1. login as admin, enter admin panel 
2. Edit an assembly
3. See the "Private uses" menu entry on the sidebar 
4. Check "Private space" and click "save"
5. See the menu entry "Private users" is missing 
6. Repeat 2,3,4,5, but this time you need to look for "Private participants"

:hearts: Thank you!
